x-pack/plugins/ml/public/application/trained_models/models_management/expanded_row.tsx

```diff
@@ -119,6 +129,41 @@ export const ExpandedRow: FC<ExpandedRowProps> = ({ item }) => {
     services: { share },
   } = useMlKibana();
 
+  useEffect(
+    function updateDeploymentState() {
+      (async function () {
+        const { nodes, ...deploymentStats } = stats.deployment_stats ?? {};
+
+        if (!isPopulatedObject(deploymentStats)) return;
+
+        const result = formatToListItems(deploymentStats)!;
+
+        const items: EuiListGroupItemProps[] = [];
+        for (const v of nodes!) {
```

Comment:
changing this to a `Promise.all` would make it a bit faster

Reply:
updated in 69a716075d134e1b51a420ee4208db4a5d13f4aa